examples/b2c/checkout/integrations/classes/B2BTaxSample.cls

```diff
@@ -11,11 +11,12 @@ global class B2BTaxSample implements sfdc_checkout.CartTaxCalculations {
             // We need to get the ID of the delivery group in order to get the DeliverTo info.
             Id cartDeliveryGroupId = [SELECT CartDeliveryGroupId FROM CartItem WHERE CartId = :cartId][0].CartDeliveryGroupId;
             CartDeliveryGroup deliveryGroup = [SELECT DeliverToState, DeliverToCountry FROM CartDeliveryGroup WHERE Id = :cartDeliveryGroupId][0];
+            String taxLocaleType = [SELECT TaxLocaleType FROM WebCart WHERE Id = :cartId][0].TaxLocaleType;
```

Comment:
We will soon need to change this field to TaxType. When cart team provides it.

Reply:
OK .. I'l leave it as is for now.